src/output/dvs_to_file.cpp

```diff
@@ -3,11 +3,51 @@
 #include <string>
 
 #include "../aedat.hpp"
```

Comment:
Potentially better to set include path and use "output/aedat.hpp", also in lines below